src/middlewares/authorize.js

```diff
@@ -49,5 +51,19 @@ const superAdmin =async(req,res,next)=>{
       isAuthorized.isNotAuthorized('Access denied', res);
     }
   }
-
-export { isSuperAdmin , adminUser, superAdmin }
+ 
+  
+const  isRequester = async (req, res, next) => {
+   try{
+      const user = await Users.findOne({where:{id:req.user.id}})
+      if(user.roleId !== 4){
```

Comment:
What exactly are you checking here?

Reply:
This is requester middleware to check if the logged in user is a requester and I have used it to make sure only requester are the only one who make likes